lib/nga.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+function handle_error() {
+    echo "Script exited with status ${2} at line ${1}"
+    if [ -z "${savefiles}" ] && [ -d $tmpprefix ]; then rm -f $tmpprefix*; fi
+    if [ -z "${savefiles}" ]  && [ -d $tmpprefix ]; then rm -f $outprefix*; fi
+}
+
+trap 'handle_error ${LINENO} $?' ERR
+
+# cause the script to fail if any commands exist with non-zero status
+set -e
+
+if [[ -z "$1" || -z "$2" || -z "$3" ]]; then
+    echo "Usage:tiff_to_pyramid.bash <tmpdir> <full path to image source> <full path to output target> <save all working files - any value will do>";
```

Comment:
Usage message doesn't match script name.

Reply:
It feels like this and the previous comment can be resolved by renaming the script `tiff_to_pyramid.bash`.